javascript/operations.js

```diff
@@ -329,11 +329,17 @@ export default {
   redirectTo: operation => {
     before(window, operation)
     operate(operation, () => {
-      let { url, action } = operation
+      let { url, action, turbo } = operation
       action = action || 'advance'
-      if (window.Turbo) window.Turbo.visit(url, { action })
-      if (window.Turbolinks) window.Turbolinks.visit(url, { action })
-      if (!window.Turbo && !window.Turbolinks) window.location.href = url
+      if (typeof turbo === 'undefined') turbo = true
+      
+      if (turbo) {
+        if (window.Turbo) window.Turbo.visit(url, { action })
+        if (window.Turbolinks) window.Turbolinks.visit(url, { action })
+        if (!window.Turbo && !window.Turbolinks) window.location.href = url
```

Comment:
I think if you `return` early after either a turbo or turbolinks visit, you could do away with the `else` branch

Reply:
If you return early, there will not be `after` callbacks fired.

Reply:
Even when just returning early from this callback?

Reply:
Oh, I see what you mean. That makes good sense. 👍